src/_guides/libraries/library-tour.md

```diff
@@ -2487,68 +2459,68 @@ To begin, you need to *reflect* on a class or object to get its
 
 Reflect on a Type to get its ClassMirror.
 
-<!-- library-tour/mirrors/bin/main.dart -->
+<?code-excerpt "test/library_tour/mirrors_test.dart (ClassMirror)" replace="/ \/\/line-br.*//g"?>
 {% prettify dart %}
 ClassMirror mirror = reflectClass(Person);
 
-assert('Person' ==
-    MirrorSystem.getName(mirror.simpleName));
+assert('Person' == MirrorSystem.getName(mirror.simpleName));
 {% endprettify %}
 
 You can also call `runtimeType` to get a Type from an instance.
 
-<!-- library-tour/mirrors/bin/main.dart -->
+<?code-excerpt "test/library_tour/mirrors_test.dart (runtimeType)"?>
 {% prettify dart %}
 var person = new Person('Bob', 'Smith', 33);
 ClassMirror mirror = reflectClass(person.runtimeType);
-assert('Person' ==
-    MirrorSystem.getName(mirror.simpleName));
+assert('Person' == MirrorSystem.getName(mirror.simpleName));
 {% endprettify %}
 
 Once you have a ClassMirror, you can get a class's constructors, fields,
 and more. Here is an example of listing the constructors of a class.
 
-<!-- library-tour/mirrors/bin/main.dart -->
+<?code-excerpt "test/library_tour/mirrors_test.dart (showConstructors)"?>
 {% prettify dart %}
-showConstructors(ClassMirror mirror) {
+void showConstructors(ClassMirror mirror) {
   var constructors = mirror.declarations.values
       .where((m) => m is MethodMirror && m.isConstructor);
 
   constructors.forEach((m) {
-    print('The constructor ${m.simpleName} has '
-          '${m.parameters.length} parameters.');
+    MethodMirror mm = m as MethodMirror;
+    print('The constructor ${mm.simpleName} has '
+        '${mm.parameters.length} parameters.');
   });
 }
 {% endprettify %}
 
 Here is an example of listing all of the fields declared by a class.
 
-<!-- library-tour/mirrors/bin/main.dart -->
+<?code-excerpt "test/library_tour/mirrors_test.dart (showFields)"?>
 {% prettify dart %}
-showFields(ClassMirror mirror) {
+void showFields(ClassMirror mirror) {
   var fields = mirror.declarations.values
       .where((m) => m is VariableMirror);
 
-  fields.forEach((VariableMirror m) {
-    var finalStatus = m.isFinal ? 'final' : 'not final';
-    var privateStatus = m.isPrivate ?
-        'private' : 'not private';
-    var typeAnnotation = m.type.simpleName;
+  fields.forEach((m) {
+    final v = m as VariableMirror;
+    var finalStatus = v.isFinal ? 'final' : 'not final';
+    var privateStatus =
+        v.isPrivate ? 'private' : 'not private';
+    var typeAnnotation = v.type.simpleName;
 
-    print('The field ${m.simpleName} is $privateStatus ' +
-          'and $finalStatus and is annotated as ' +
-          '$typeAnnotation.');
+    print('The field ${v.simpleName} is $privateStatus ' +
+        'and $finalStatus and is annotated as ' +
+        '$typeAnnotation.');
   });
-}{% endprettify %}
+}
+{% endprettify %}
```

Comment:
FYI, the updates from the code example above are from https://github.com/dart-lang/site-www/blob/master/examples/library_tour/mirrors/bin/main.dart